.env.mainnet

```diff
@@ -5,8 +5,8 @@ SMS_RETRIEVER_APP_SIGNATURE=bU9E4ctGtIW
 DEV_SETTINGS_ACTIVE_INITIALLY=false
 FIREBASE_ENABLED=false
 SHOW_TESTNET_BANNER=false
-APP_BUNDLE_ID=org.mento.app
-APP_STORE_ID=1520414263
+APP_BUNDLE_ID=xyz.mobilestack.mento
```

Comment:
should other env's have a similar change? I think this may also need a chance in the cab settings.